app/services/cards_service.rb

```diff
@@ -0,0 +1,38 @@
+class CardsService
+  class << self
+    def parsed_html?(user_id, block_id, url, css_original, css_translated)
+      parsed = false
+
+      if block_id && user_id
+        html = Nokogiri::HTML(open(url))
+
+        original_texts = html.css(css_original).map { |original| original.content }
+        translated_texts = html.css(css_translated).map { |translated| translated.content }
+
+        if original_texts.size == translated_texts.size && original_texts.any?
+          if original_texts[0] != translated_texts[0]
+            Card.delay.create_cards_from_html(user_id, block_id,
+                                              original_texts, translated_texts)
+
+            parsed = true
+          end
+        end
+      end
+
+      parsed
+
+    rescue Exception
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?